diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/LineLength.kt

```diff
@@ -74,6 +109,191 @@ class LineLength : Rule("line-length") {
+                    return
+                }
+                else -> parent = parent.treeParent
+            }
+        } while (parent.treeParent != null)
+    }
+
+    private fun fixComment(wrongNode: ASTNode, configuration: LineLengthConfiguration){
+        var node = wrongNode
+        do {
+            val commentText = node.text
+            var indexLastSpace = configuration.lineLength.toInt()
+            while (commentText[indexLastSpace] != ' ') {
+                --indexLastSpace
+                if (indexLastSpace == 0)
+                    return
+            }
+            val nodeText = "//${commentText.substring(indexLastSpace, commentText.length)}"
+            val newNode = LeafPsiElement(EOL_COMMENT, nodeText)
+            node.treeParent.addChild(LeafPsiElement(EOL_COMMENT, commentText.substring(0, indexLastSpace)), node)
+            node.treeParent.addChild(PsiWhiteSpaceImpl("\n"), node)
+            node.treeParent.addChild(newNode, node)
+            node.treeParent.removeChild(node)
+            node = newNode
+        } while (node.text.length <= configuration.lineLength)
+    }
+
+    private fun fixCondition(wrongNode: ASTNode, configuration: LineLengthConfiguration, grand: ASTNode,
+                             leftInitOffset: Int = -1, isProperty: Boolean = false) {
+        var leftOffset = if (leftInitOffset < 0){
+            val positionByOffset = calculateLineColByOffset(grand.treeParent.text)
+            positionByOffset(wrongNode.firstChildNode.psi.textOffset).second
+        } else leftInitOffset
+        val binList = mutableListOf<ASTNode>()
+        if (isProperty)
+            dfsForProperty(wrongNode, binList)
+        else
+            searchBinaryExpression(wrongNode, binList)
+        lateinit var binaryText: String
+        binList.forEachIndexed { index, astNode ->
+            if (index == 0){
+                binaryText = astNode.text
+            } else {
+                val operationRef = astNode.treeParent.findChildByType(OPERATION_REFERENCE)!!
+                if (operationRef.treePrev.elementType == WHITE_SPACE)
+                    binaryText += operationRef.treePrev.text
+                binaryText += operationRef.text
+                if (operationRef.treeNext.elementType == WHITE_SPACE)
+                    binaryText += operationRef.treeNext.text
+                binaryText+= astNode.text
+                if (leftOffset + binaryText.length > configuration.lineLength) {
+                    val commonParent = findCommonParent(astNode, binList[index-1])!!
+                    val newLine = PsiWhiteSpaceImpl("\n")
+                    commonParent.addChild(newLine, commonParent.findChildByType(OPERATION_REFERENCE)!!.treeNext)
+                    leftOffset = BINARY_OFFSET
+                    binaryText = astNode.text
+                }
+            }
+        }
+    }
+
+    private fun findCommonParent(firstNode: ASTNode ,secondNode: ASTNode): ASTNode?{
```

Comment:
In which cases this method returns `null`? I think the only place where you use this method you also apply `!!`?